src/modules/reporting/components/reporting-dispute-progress/reporting-dispute-progress.styles.less

```diff
@@ -16,37 +16,58 @@
 }
 
 .ReportingDisputeProgress__dispute-label {
+  letter-spacing: 0.3px;
+  margin-left: 1rem;
+  white-space: nowrap;
+}
+
+.ReportingDisputeProgress__dispute-label-total-rep-text {
+  color: @color-lightergray;
+  font-size: @font-size-medium;
+  font-weight: 500;
+}
+
+.ReportingDisputeProgress__dispute-label-break {
   color: @color-lightgray;
-  font-size: 0.6875rem;
+  font-size: @font-size-medium;
   font-weight: 500;
-  letter-spacing: 0.3px;
-  line-height: 0.6875rem;
-  margin-left: 1.5rem;
+  margin-left: 0.2em;
+}
+
+.ReportingDisputeProgress__dispute-label-goal-text {
+  color: @color-lightgray;
+  font-size: @font-size-small;
+  font-weight: 500;
+  line-height: 1em;
+  margin-left: 0.2em;
   white-space: nowrap;
 }
 
 .ReportingDisputeProgress__dispute-tentative {
   &:extend(.ReportingDisputeProgress__dispute-label);
 
   color: @color-lighter-purple;
+  font-size: @font-size-small;
   margin-left: 1.5rem;
   text-transform: uppercase;
 }
 
 .ReportingDisputeProgress__graph-current {
-  background: @color-gray;
+  background: #4f4761;
   display: flex;
```

Comment:
Use color type instead of hex.

Reply:
i think this is a one use color

Reply:
ok, we'll keep it the way it is then. :)